### DISCUSSION TITLE
SSL Error on running `npm run dev` for `doc` repo locally

### DISCUSSION THREAD

**Discussion by satyajitnayk** [handlebars-lang/handlebars.js#1944] [Q&A] [answered]
I have forked https://github.com/handlebars-lang/docs repo & tried running locally, but the following error comes up.
```
Error: error:0308010C:digital envelope routines::unsupported
    at new Hash (node:internal/crypto/hash:71:19)
    at Object.createHash (node:crypto:133:10)
```
```
 opensslErrorStack: [ 'error:03000086:digital envelope routines::initialization error' ],
  library: 'digital envelope routines',
  reason: 'unsupported',
  code: 'ERR_OSSL_EVP_UNSUPPORTED'
```

<img width="1195" alt="image" src="https://github.com/handlebars-lang/handlebars.js/assets/32722867/02f3aeb3-949e-4b9e-b06e-2dc0d40d600c">

**machine specification:**
OS: Mac OS M1 Air
Version: Ventura 13.3.1 (a) 
Node Version: v18.12.1

Any help in this reagrds is highly appreciated from any one who have contributed to https://github.com/handlebars-lang/docs repo

**Comment by jaylinski** [comment, marked as answer]
I think the docs-repo doesn't work with Node >= v17 yet, because we use an outdated webpack version: https://github.com/webpack/webpack/issues/14532

We should probably upgrade to `vitepress` (right now we use `vuepress`).

**Reply by satyajitnayk** [reply]
Thanks @jaylinski again for help.
